src/baldr/edgeinfo.cc

```diff
@@ -313,11 +313,17 @@ std::string EdgeInfo::level_ref() const {
 json::MapPtr EdgeInfo::json() const {
   json::MapPtr edge_info = json::map({
       {"way_id", static_cast<uint64_t>(wayid())},
-      {"mean_elevation", static_cast<uint64_t>(mean_elevation())},
       {"bike_network", bike_network_json(bike_network())},
       {"names", names_json(GetNames())},
       {"shape", midgard::encode(shape())},
   });
+  // add the mean_elevation depending on its validity
+  const auto elev = mean_elevation();
+  if (elev == kNoElevationData) {
+    edge_info->emplace("mean_elevation", nullptr);
+  } else {
+    edge_info->emplace("mean_elevation", static_cast<int64_t>(elev));
```

Comment:
hm.. this is still missing a possible conversion to miles I just realized

Reply:
that's actually a rhetorical question.. a request with or without `"directions_options":{"units":"miles"}` has no impact. I'll PR that separately

Reply:
Yes, locate was designed to just regurgitate exactly what is in the tile data back to the requester. It's useful for programmatic things not meant to be used directly by a human. For this reason I don't think we should at all do units conversions. The api docs should just say everything is metric. I could be convinced it makes sense to care about units though. Seems you feel like it should. Could you maybe make a case for it just so I could understand the use case you have in mind?

Reply:
I wasn't sure myself if that's supposed to be supported by the locate api, then I saw the docs are explicitly using `miles`  in the example so I figured it should convert. very happy not to go through the process tbh, won't make an argument here;)